src/api/app/views/webui2/webui/request/_show_side_links.html.haml

```diff
@@ -0,0 +1,45 @@
+%ul.list-unstyled
+  %li
+    %i.fas.fa-user.text-success
+    Created
+    by #{user_with_realname_and_icon(request_creator, no_icon: true, short: true)} #{fuzzy_time(request_created_at)}
+  %li
+    %i.fa.fa-info-circle.text-info{ title: 'Info' }
+    In
+    state #{link_to(request_state, { anchor: 'request_history' }, style: "color: #{request_state_color(request_state)};")}
```

Comment:
Please put the Ruby code on another line, you won't have to interpolate it in the string.

We should use classes for the color instead of `style`